### PR TITLE
feature/reflex

### PR DESCRIPTION
**Pull request by cutoffthetop**
# PR Context

- switch tech stack to python/reflex

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,27 +51,49 @@ components of the MEx project are open-sourced under the same license as well.
...
+- update package dependencies using `pdm update-all`
+- update github actions in `.github/workflows/*.yml` manually
 
 ### creating release
 
-- update version in `package.json` and `CHANGELOG.md`
+- update version in `pyproject.toml` and `CHANGELOG.md`
 - commit update `git commit --message "..."`
 - create a tag `git tag ...`
 - push `git push --follow-tags`
```
Comment (rababerladuseladim) [inline README.md]:
```suggestion
- run `pdm release RULE` to release a new version where RULE determines which part of
  the version to update and is one of `major`, `minor`, `patch`.
```

File: mex/editor/transform.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    exclude: set[str] | None = None,
+    by_alias: bool = True,
+    exclude_unset: bool = False,
+    exclude_defaults: bool = False,
+    exclude_none: bool = False,
+    custom_encoder: dict[Any, Callable[[Any], Any]] = JSON_ENCODERS,
+) -> Any:
+    """Convert any object into python primitives compatible with JSONification."""
+    if isinstance(obj, BaseModel):
+        return obj.__pydantic_serializer__.to_python(
```
Comment (rababerladuseladim) [inline mex/editor/transform.py:32]: Why not:
```suggestion
        return obj.model_dump(
```
?

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+unit = { cmd = "pdm run pytest -m 'not integration'" }
+test = { cmd = "pdm run pytest --numprocesses=auto --dist=worksteal" }
+all = { composite = ["install-all", "lint", "test", "doc"] }
+
+[tool.pydantic-mypy]
+warn_untyped_fields = true
+
+[tool.pyright]
+venvPath = "."
+venv = ".venv"
```
Comment (rababerladuseladim) [inline pyproject.toml]: Who is using pyright and why?

**Review by rababerladuseladim — CHANGES_REQUESTED** [review]
biutiful. Thank you very much for this beautiful new editor prototype!

File: pyproject.toml (hunk 3 above)
Comment (cutoffthetop) [reply pyproject.toml]: ups, not me. 1188acc

File: mex/editor/transform.py (hunk 2 above)
Comment (cutoffthetop) [reply mex/editor/transform.py:32]: added a reason here: 82e2777

**Pull request merged**